docs/content/dagster-cloud/deployment/serverless.mdx

```diff
@@ -76,10 +76,19 @@ dagster-cloud configure
 
 You can also configure the `dagster-cloud` tool noninteractively; see [the CLI docs](/dagster-cloud/developing-testing/dagster-cloud-cli#environment-variables-and-cli-options) for more information.
 
+Add 'dagster-cloud' as a dependency to `my-dagster-project/setup.py`:
+
+```
+    install_requires=[
+        "dagster",
+        "dagster-cloud",    # add this line
+        "dagster-airbyte",
```

Comment:
```suggestion
install_requires=[
    "dagster",
    "dagster-cloud",    # add this line
    "dagster-airbyte",
```

🔧 In this case, the code should be flush-left. You might need to run `make mdx-format` locally after making this change, or BK could throw a fit.